### PR TITLE
DOCSP-32548: Breaking changes consolidation

### PR DESCRIPTION
**Pull request by norareidy**
# Pull Request Info

[PR Reviewing Guidelines](https://github.com/mongodb/docs-node/blob/master/REVIEWING.md)

JIRA - <https://jira.mongodb.org/browse/DOCSP-32548>
Staging - <https://docs-mongodbcom-staging.corp.mongodb.com/node/docsworker-xlarge/DOCSP-32548-consolidate-breaking/whats-new/>

## Self-Review Checklist

- [x] Is this free of any warnings or errors in the RST?
- [x] Did you run a spell-check?
- [ ] Did you run a grammar-check?
- [x] Are all the links working?

### CODE REVIEW

File: source/whats-new.txt [hunk 1]
```diff
@@ -48,55 +48,11 @@ Learn what's new in:
...
-     Operations guide for more information.
-   
-   - Raised the optional ``kerberos`` dependency minimum version to 2.0.1 and
-     removed support for version 1.x.
-
-   - Raised the optional ``zstd`` dependency minimum version to 1.1.0.
-
-   To learn more about the breaking changes included in this release, see the
-   `v6.0.0 Release Highlights <https://github.com/mongodb/node-mongodb-native/releases/tag/v6.0.0>`__.
+   This driver version has breaking changes. For a list of these changes, see
```
Comment (rustagir) [inline source/whats-new.txt:53]:
```suggestion
   This driver version introduces breaking changes. For a list of these changes, see 
```

Comment (rustagir) [reply source/whats-new.txt:53]: applies elsewhere, too

File: source/whats-new.txt [hunk 2]
```diff
@@ -48,55 +48,11 @@ Learn what's new in:
...
-     :ref:`node-connect-socks`.
-     
-   - If you start a session on a client, then pass that session to a
-     different client, the driver throws an error when you
-     perform any operations in the session.
-   
-   - The ``includeResultMetadata`` option for compound operation methods
-     is ``false`` by default. See the :ref:`Built-in
-     Methods <node-compound-operations-builtin>` section of the Compound
-     Operations guide for more information.
```
Comment (rustagir) [inline source/whats-new.txt:91]: S: for certain entries, a page is linked in this note, but not in the breaking changes guide. Could you go through and make sure that additional information is linked in the breaking changes guide wherever they have been linked in these admonitions?

**Review by rustagir — CHANGES_REQUESTED** [review]
Requesting changes so I can see the changes after you add the links back to the breaking changes guide!

File: source/whats-new.txt (hunk 2 above)
Comment (norareidy) [reply source/whats-new.txt:91]: Thanks for catching this! I've added the links to installing the socks package (node-connect-socks) and information about compound operations (node-compound-operations-builtin).

**Review by rustagir — APPROVED** [review]
lgtm!

**Pull request merged**